sdk/python/express_relay/svm/limo_client.py

```diff
@@ -239,6 +242,8 @@ async def take_order_ix(
             ixs.extend(instructions["fill_ixs"])
             close_wsol_ixns.extend(instructions["close_ixs"])
             taker_output_ata = instructions["ata"]
+
+            intermediary_output_token_account = self.get_intermediary_token_account_pda(PROGRAM_ID, order["address"])
```

Comment:
Checking the JS sdk it seems like you also need to set `maker_output_ata` to None in this case?